packages/orangutan/src/index.js

```diff
@@ -1,5 +1,5 @@
 import dualChannel from '@twreporter/dual-channel'
-
+import scrollableImage from '@twreporter/scrollable-image'
```

Comment:
just found that `@twreporter/scrollable-image` has not added to `packages.json` of `@twreporter/orangutan` yet